pytests/helpers/session.py

```diff
@@ -204,7 +210,7 @@ def wait_for_disconnect(self, close=True):
         if close:
             self.timeline.close()
 
-    def wait_for_termination(self, expected_returncode=0):
+    def wait_for_termination(self, expected_returncode):
```

Comment:
Since we now have `expected_returncode` as a property, I think it should be removed from the arguments for all these functions here that have it currently - tests should just set the property as needed. I don't think there are any other tests that pass it explicitly at the moment, so it should be an easy change.